CodeMaid/Logic/Reorganizing/CodeReorganizationManager.cs

```diff
@@ -364,9 +364,8 @@ private void RecursivelyReorganize(IEnumerable<BaseCodeItem> codeItems, ICodeIte
 
             // Get the items in their current order and their desired order.
             var currentOrder = GetReorganizableCodeItemElements(codeItems);
-            var desiredOrder = currentOrder.OrderBy(CodeItemTypeComparer.CalculateNumericRepresentation)
-                                           .ThenBy(x => Settings.Default.Reorganizing_AlphabetizeMembersOfTheSameGroup ? (object)x.Name : (object)x.StartOffset)
-                                           .ToList();
+            var desiredOrder = new List<BaseCodeItemElement>(currentOrder);
+            desiredOrder.Sort(new CodeItemTypeComparer(Settings.Default.Reorganizing_AlphabetizeMembersOfTheSameGroup));
```

Comment:
I'm quite confident. Although List.Sort isn't a stable sort, I don't think we'll ever return two things as equal as we use StartOffset as a tiebreaker (and if that was the same it wouldn't matter switching them around ;-) )

The ThenBy part is handling inside the comparer, as it was already there for the Spade tool. Since they use different settings the flag to switch on name sorting is passed into the constructor.